snap/naming/validate.go

```diff
@@ -163,6 +163,17 @@ func ValidateSocket(name string) error {
 	return nil
 }
 
+// ValidateLabel can be used to check valid labels in interfaces.
+// These labels are used to match plugs with slots, and although they
+// could be arbitrary strings it is nice to keep naming consistent
+// with what we do for snap names.
+func ValidateLabel(name string) error {
```

Comment:
mmh, calling things here label is a bit confusing because as I said PlugInfo.Label and SlotInfo.Label exist

Reply:
I've switched to using "tag" to differentiate now